.github/workflows/build_all.yml

```
@@ -17,6 +23,8 @@ jobs:
     env:
       TEST_BUILD_ALL: 1
       TEST_FATAL_WARNINGS: ${{ github.event.inputs.fatal_warnings }}
+      TEST_REPOSITORY: ${{ github.event.inputs.repository }}
+      TEST_BRANCH: ${{ github.event.inputs.branch }}
```

These should fall back to the canonical values.

```suggestion
      TEST_REPOSITORY: ${{ github.event.inputs.repository || 'https://github.com/mesonbuild/meson' }}
      TEST_BRANCH: ${{ github.event.inputs.branch || 'master' }}
```